docs/_src/api/api/reader.md

```diff
@@ -158,7 +158,7 @@ Saves the Reader model so that it can be reused at a later point in time.
 
 Use loaded QA model to find answers for a list of queries in each query's supplied list of Document.
 
-Returns list of dictionaries containing answers sorted by (desc.) probability
+Returns list of dictionaries containing answers sorted by (desc.) score
```

Comment:
Did you change those manually? The API docs will be generated automatically from the docstrings of the `FARMReader` class (see https://github.com/deepset-ai/haystack/tree/master/docs#update-docstrings)

Reply:
True. I will revert this change.